config/rbac/nutanixclustertemplate_editor_role.yaml

```diff
@@ -0,0 +1,25 @@
+# permissions for end users to edit nutanixclustertemplates.
```

Comment:
would be useful to spell out in a top-of-file comment that the contents of the file were autogenerated.

Reply:
if needed we can take it in a separate PR since its common to all but as of now its consistent with all others